CONTRIBUTORS

```diff
@@ -1,3 +1,4 @@
 andreimuntean
 AndrewBLC
 MisterDoom
+alexdmtr
```

Comment:
Nit: I'd keep this list sorted alphabetically.

Reply:
Agree, fixed :+1: